### ISSUE TITLE
Add get_gateway_id() to gateway abstract

### ISSUE THREAD

**Issue by helgatheviking** [Charitable/Charitable#554]
This method appears in all the different gateways I see, but it's always the same. Why isn't it in the abstract instead?

```
		/**
		 * Returns the current gateway's ID.
		 *
		 * @since  1.0.3
		 *
		 * @return string
		 */
		public static function get_gateway_id() {
			return self::ID;
		}
```

**Comment by ericnicolaas** [comment]
Because of PHP 5.2 :(

`self::ID` will return the abstract's gateway ID if the method isn't defined in the gateways. `static::ID` returns the correct thing, but isn't available in 5.2. 

Closing this as wontfix for now.

**Comment by helgatheviking** [comment]
Freaking PHP5.2. Wish WordPress core would at least require 5.6 already. Sheesh. Thanks for the explanation.
